legacy/src/com/futurewei/alcor/controller/resourcemgr/physical/nodemgmt/DataCenterConfigLoader.java

```diff
@@ -16,17 +16,26 @@
 
 package com.futurewei.alcor.controller.resourcemgr.physical.nodemgmt;
 
+<<<<<<< HEAD:legacy/src/com/futurewei/alcor/controller/resourcemgr/physical/nodemgmt/DataCenterConfigLoader.java
```

Comment:
This changes legacy codes. Could we revert?

Reply:
Class name is changed (DataCenterConfigLoader => NodeConfigLoader ) to avoid the conflict.

Reply:
Okay but the conflict is still not resolved..